dev/docker_data/default_pipeline.yml

```diff
@@ -453,11 +486,10 @@ regWithSkull :  [1]
 runSegmentationPreprocessing :  [1]
 
 
-# ANTs Prior-based Segmentation workflow that has shown optimal results for non-human primate data.
-# Generate white matter, gray matter, CSF masks based on antsJointLabelFusion
-# If you'd like to set ANTs Prior-based Segmentation as your segmentation option,
-# please set Template based segmentation as 'None' and runSegmentationPreprocessing as [0].
-ANTs_prior_based_segmentation : [0]
+# segmentation method options: ['FSL', 'ANTs-Prior-Based', 'Template-Based']
```

Comment:
This segmentation refactoring part hasn't finished. I only refactored ANTs prior-based segmentation. Do I need to revert changes or keep refactoring other parts?

Reply:
Should keeping what you've done so far break anything? If not, I think we can keep what's done and refactor the rest after 1.8.0

Reply:
Yep, good idea!